src/convertToObject.js

```diff
@@ -6,7 +6,40 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
+  const cleanedSourceString = sourceString.replace(/\t+/, ' ').trim();
+  const styleStrings = cleanedSourceString.split(';');
+  let currentProperty = '';
+  let currentValue = '';
+
+  styleStrings.forEach((styleString) => {
+    if (styleString.trim() === '') {
```

Comment:
You don't need to compare with an empty string, just check if a value exists https://developer.mozilla.org/en-US/docs/Glossary/Falsy